# Related references (COVID: S1RBD-hACE2 complex)

[![hackmd-github-sync-badge](https://hackmd.io/H7Wj8HWzQiiisXRIxP_lGQ/badge)](https://hackmd.io/H7Wj8HWzQiiisXRIxP_lGQ)

| \# |  Variants | PDB | Exp./MD Method | Analysis | Keypoints |
| -------- | -------- | -------- | -------- | -------- |  -------- |
|[1](https://doi.org/10.3390/molecules27072370)| WT, Gamma | 6M0J, 7NXB | gromacs (Amber99sb-ildn + TIP3P, NaCl 0.150 M, 300 K, 100 ns) | MM-PBSA, RMSD, RMSF, | Mutations E484K and K417T compensate each other in terms of binding affinity, while the mutation N501Y promotes a more convoluted effect. This effect consists in the adoption of a cis configuration in the backbone of residue Y495 within the RBD, which in turn promotes polar interactions with the hACE-2 receptor. |
|[2](https://doi.org/10.1016/j.ijbiomac.2022.01.059)| WT, Omicron (B.1.1.529) | 6m0j, (7c2l) | AMBER20 (FF19SB + OPC, 300K, 500 ns) | HADDOCK, RMSD, RMSF, MM-PBSA, H-bonds counts | Asn417, Ser446, Arg493, Arg498 helps the new variants to bind more strongly than the wild type. B.1.1.529 displayed a stable dynamic and reported a strong correlation with the previous reports where stability increasing mutations in the RBD were reported to enhance the binding. |
|[3](https://doi.org/10.1016/j.cell.2022.01.001)|WT, delta, omicron (B.1.1.529) | 7v7v, 6lzg, 7wbl, 7wbp, 7wbq | cryo-EM, X-ray | SPR Analysis | Omicron, delta, and prototype SARS-CoV-2 RBDs show similar binding strength to hACE2. The complexes of SARS-CoV-2-RBD with hACE2 for omicron and delta variants were resolved. The roles of key residues in the omicron RBD for receptor recognition were identified. |
|[4](https://doi.org/10.1038/s41467-022-31276-6)| WT, alpha, beta, gamma, delta, omicron (BA.1)| 6LZG |cryo-EM| SPR analysis | We identified S494 of SARS-COV-2 PT-RBD as an important residue in the eqACE2/SARS-COV-2 PT-RBD interaction and found that N501Y, the commonly recognized enhancing mutation, attenuated the binding affinity with eqACE2. |
|[5](https://doi.org/10.3390/biology11050797) |  delta, omicron (BA.1, BA.2) | 7WBP, 7WBQ | Gromacs (CHARMM36m +  TIP3, 0.15 M KCl, 303.15 K, 1 atm, 100 ns) | HDOCK, Prodigy, RMSD, RMSF, Rg | According to docking interactionenergies and interface-interaction analysis, Omicron BA2 exhibited a higher affinity forhACE2 than Omicron BA1. From molecular dynamics simulations, it was concluded thatthe strong interaction between protein–protein complexes persisted.|
|[6](https://doi.org/10.1021/acs.jcim.2c00100)|WT, delta, omicron|6m0j, 7v8b, 7t9l|AlphaFold, NAMD (CHARMM36m, 310 K, 60ns) |PIPER (docking), RMSD, FMO calculations, pair interaction energy (PIE)|S375F was found to be a mutation that majorly changed the binding affinity of the spike protein to hACE2 and the eight monoclonal antibodies. |
|[7](https://doi.org/10.1371/journal.pone.0266844)| WT, omicron (B.1.529.1)| 6m0j, 7n1u | GROMACS (CHARMM36 + TIP3P, 0.15mM NaCl, 310.15K, 200ns) | PRODIGY, HDOCK (docking), RMSD, RMSF, MM-GBSA|The RBD of the Omicron variant had a  higher affinity than the reference. Moreover, all-atom molecular dynamics simulations concluded that the RBD of the Omicron variant exhibits a more dispersed interaction network since mutations resulted in an increased number of hydrophobic interactions and hydrogen bonds with hACE2.|
|[8](https://doi.org/10.1016/j.cell.2022.06.023)|alpha, beta, gamma, delta, omicron(BA.1, BA.1.1, BA.2, BA.3)| 7XAZ, 7XB0, 7XB1, 6LZG, 7WBP | X-ray, GROMACS (CHARMM36 + TIP3P, 300 K, 200 ns) | MM-PBSA, SPR assay | The complex structures of hACE2 with RBDs of BA.1.1, BA.2 and BA.3 reveal that the higher hACE2 binding affinity of BA.2 than BA.1 is related to the absence of the G496S mutation in BA.2. The R346K mutation in BA.1.1 majorly affects the interaction network in the BA.1.1 RBD/hACE2 interface through long-range alterations and contributes to the higher hACE2 affinity of the BA.1.1 RBD than the BA.1 RBD. |
|[9](https://doi.org/10.1002/jmv.27526)| WT, delta, omicron (B.1.1.529)| 6M0J | - |HEX docking program, I-Mutant3.0, PredyFlexy, sorting intolerant from tolerant (SIFT) tool, VarSite |Q493R, N501Y, S371L, S373P, S375F, Q498R, and T478K mutations contribute significantly to high binding affinity with human ACE2. A high proportion of hydrophobic amino acids, such as leucine and phenylalanine, are located within the protein's core and are required for structural stability. |
|[10](https://doi.org/10.1016/j.bbrc.2022.06.064)|WT|6m17, 7df4 | Mutabind2, Dynamut2, CABS-flex 2.0|Rg, RMSF|SARS-CoV-2 interface region (Residue 417–505) interacts with the human ACE2 receptor. 52 energetically favorable Spike mutations at the interface were identified. 36 mutations out of 52 significantly enhance the stabilization of the Spike-ACE2 complex. Identified mutations K417Y and E484A were reported to be variants of concern for SARS-CoV-2: Omicron.|
|[11](https://doi.org/10.1038/s41586-022-04980-y)|WT, omicron (BA.1, BA.2, BA.3, BA.2.13, BA.2.12.1, BA.4, BA.5) |7WGB| cryo-EM, gromacs (300K, 1 bar, 10 ns)| SPR, Biolayer Interferometry |Most of these cross-reactive NAbs are heavily escaped by L452Q, L452R and F486V. Nevertheless, these NAbs are largely escaped by BA.2/BA.4/BA.5 due to D405N and F486V. BA.4/BA.5 with the S371F, D405N and R408S mutations would undermine most broad sarbecovirus NAbs.｜
|[12](https://doi.org/10.3390/ijms23063409)|WT, omicron|7A91|AMBER 18.0 (f14SB + TIP3P, 300 K, 1 atm, 100 ns)|RMSF, Rg, MM-GBSA (last 10 ns)|We have observed extra salt bridges between the RBM-R493 -- D30-ACE2, the RBM-R493 -- E35-ACE2, and the RBM-R493 -- D38-ACE2 in the Omicron variant. RBM’s mutated resides formed some additional hydrogen bonding and pi-stacking interactions, which could further enhance hACE2 binding. |
|[13](https://doi.org/10.1038/s41422-022-00672-4)|WT, Omicron (BA.1, BA.2)|7WPA|cryo-EM|Thermal shift assay (TSA), biolayer interferometry|Omicron BA.2 spike trimer exhibits 11-fold and 2-fold higher potency in binding to human ACE2 than the spike trimer from the wildtype (WT) and Omicron BA.1 strains. |
|[14](https://doi.org/10.1002/jmv.27927)|WT, omicron (BA.1, BA.1.1, BA.2, BA.3) | 6M0J, 7T9L|homology modeling |Hawkdock and cluspro docking, Ramachandran plot, Errat plot, TM‐score, RMSD, adaptive Poisson–Boltzmann Solver program, PredictSNP|The Omicron sub‐variants (BA.1.1, BA.2 and BA.3) are likely more transmissible than omicron (BA.1) and Delta. |
|[15](https://doi.org/10.1021/acs.jpcb.2c01048)|WT, omicron|6LZG | gromacs (AMBER19SB + GLYCAM06j + OPC, 0.15 M NaCl, 300 K and 1 atm, 200 ns) |MM-PBSA, Hydrogen Bond, Side chain Contact|The electrostatic interaction was found to rule the stability of the viral RBD–hACE2 PD complex. We demonstrated that N440K, T478K, E484A, Q493R, and Q498R mutations play a crucial role in the high binding affinity of Omicron to human cells. Glycans have a little effect on the binding affinity of the WT RBD to hACE2.|
|[16](https://doi.org/10.1038/s41392-021-00863-2)|WT, delta, omicron|6VW1, 7VVS, 7N1X, (7C01, 7MMO, 7CM4, 7CHH, 7L3N)|gromacs (pH 7.4, 150 mM NaCl, amberff14SB + TIP3P, 1.0 bar, 300 K, 200 ns), Binding ELISA |MM/GBSA| Omicron variant possesses comparable binding affinity to human ACE2 in comparison with the wild type SARS-CoV-2, but much weaker binding affinity than Delta variant.|
|[17](https://doi.org/10.1016/j.csbj.2022.02.015)|WT, delta, epsilon, kappa, omicron|7KMB, 7MJN|Amber20 (ff14SB + TIP3P, 310 K, 500 ns)|RMSF, analysis of contacts, measurement of interatomic distances, calculation of electrostatic linear interaction energy, hydrogen bond analyses, mmpbsa|We identified residue 493 in delta (glutamine) and omicron (arginine) with altered binding properties towards ACE2.|
|[18](https://doi.org/10.1126/science.abn7760)|WT, delta, omicron|7MJG, 7MJN, 7T9J, 7T9L, 7T9K|cryo-EM|SPR analysis|New salt bridges and hydrogen bonds were formed by mutated residues arginine-493, serine-496, and arginine-498 in the receptor binding domain with ACE2. These interactions appear to compensate for other Omicron mutations such as the substitution of asparagine for lysine at position 417 (K417N) that are known to reduce ACE2 binding affinity. |
|[19](https://doi.org/10.1038/s42003-021-02946-w)|WT, alpha, beta, gamma|6M0J, 7MJN, 7NXC, 7KJ5, 7LYN|Gromacs (CHARMM-27 + TIP3P, 300 K, 1 bar, 500 ns)|HDOCK, PCA, clustering algorithm (hclust, R), surface analysis (DMS), LINCS algorithm|We found that a phenylalanine mutation behaves similarly to the English (alpha) variant and may cooperate in further increasing the stability of the South African (beta) one.|
|[20](https://doi.org/10.1038/s41598-022-12479-9)|WT, alpha, delta, omicron|6M0J|AMBER16 (ff14SB + TIP3P, pH 7, 200 ns) accelerated molecular dynamics (aMD) | MMGBSA, PCA, RMSD, RMSF|Omicron RBD–ACE2 complex present similar fluctuation in comparison to S protein from WT, Delta and Alpha variants. The stabilization effect RBDOmicron–ACE2 complex is achieved manly due the substitution of uncharged residues by positively charged residues: Lys and Arg in key positions.|
|[21](https://doi.org/10.1101/2022.01.24.477633)|alpha, delta, omicron|6vsb, 6vw1|steered molecular dynamics (SMD) simulation : NAMD (CHARMM36 + TIP3P, 0.15 M KCl), microscale thermophoresis (MST) |-|Omicron mutations in RBD are associated with a five-fold higher binding affinity to ACE2 compared to the RBD of the original strain. N501Y, Q493K/R, and T478K mutations are responsible for the higher force for Omicron RBD-ACE2 dissociation. |
|[22](https://doi.org/10.1101/2021.12.29.474402)|WT, delta, omicron|6VYB, 6M0J|cryo-EM|Bio-layer interferometry, SPR |We have identified three new immune escape sites: 1) Q493R, 2) G446S and 3) S371L/S373P/S375F that confers greater resistance to five of the six classes of RBD-antibodies. |


# References list [APA format]
1. Cavani, M., Riofrío, W. A., & Arciniega, M. (2022). Molecular Dynamics and MM-PBSA Analysis of the SARS-CoV-2 Gamma Variant in Complex with the hACE-2 Receptor. Molecules, 27(7), 2370. 
2. Khan, A., Waris, H., Rafique, M., Suleman, M., Mohammad, A., Ali, S. S., ... & Wei, D. Q. (2022). The Omicron (B. 1.1. 529) variant of SARS-CoV-2 binds to the hACE2 receptor more strongly and escapes the antibody response: Insights from structural and simulation data. International Journal of Biological Macromolecules, 200, 438-448.
3. Han, P., Li, L., Liu, S., Wang, Q., Zhang, D., Xu, Z., ... & Wang, P. (2022). Receptor binding and complex structures of human ACE2 to spike RBD from omicron and delta SARS-CoV-2. Cell, 185(4), 630-640.
4. Xu, Z., Kang, X., Han, P., Du, P., Li, L., Zheng, A., ... & Gao, G. F. (2022). Binding and structural basis of equine ACE2 to RBDs from SARS-CoV, SARS-CoV-2 and related coronaviruses. Nature Communications, 13(1), 1-11.
5. Celik, I., Abdellattif, M. H., & Tallei, T. E. (2022). An Insight Based on Computational Analysis of the Interaction between the Receptor-Binding Domain of the Omicron Variants and Human Angiotensin-Converting Enzyme 2. Biology, 11(5), 797.
6. Hwang, S., Baek, S. H., & Park, D. (2022). Interaction Analysis of the Spike Protein of Delta and Omicron Variants of SARS-CoV-2 with hACE2 and Eight Monoclonal Antibodies Using the Fragment Molecular Orbital Method. Journal of chemical information and modeling, 62(7), 1771-1782.
7. Shishir, T. A., Jannat, T., & Naser, I. B. (2022). An in-silico study of the mutation-associated effects on the spike protein of SARS-CoV-2, Omicron variant. Plos one, 17(4), e0266844.
8. Li, L., Liao, H., Meng, Y., Li, W., Han, P., Liu, K., ... & Gao, G. F. (2022). Structural basis of human ACE2 higher binding affinity to currently circulating Omicron SARS-CoV-2 sub-variants BA. 2 and BA. 1.1. Cell.
9. Kumar, S., Thambiraja, T. S., Karuppanan, K., & Subramaniam, G. (2022). Omicron and Delta variant of SARS‐CoV‐2: a comparative computational study of spike protein. Journal of medical virology, 94(4), 1641-1649.
10. Mishra, P. M., Anjum, F., Uversky, V. N., & Nandi, C. K. (2022). SARS-CoV-2 Spike mutations modify the interaction between virus Spike and human ACE2 receptors. Biochemical and Biophysical Research Communications.
11. Cao, Y., Yisimayi, A., Jian, F., Song, W., Xiao, T., Wang, L., ... & Xie, X. S. (2022). BA. 2.12. 1, BA. 4 and BA. 5 escape antibodies elicited by Omicron infection. Nature, 1-3.
12. Kumar, R., Murugan, N. A., & Srivastava, V. (2022). Improved binding affinity of omicron’s spike protein for the human angiotensin-converting enzyme 2 receptor is the key behind its increased virulence. International journal of molecular sciences, 23(6), 3409.
13. Xu, Y., Wu, C., Cao, X., Gu, C., Liu, H., Jiang, M., ... & Yin, W. (2022). Structural and biochemical mechanism for increased infectivity and immune evasion of Omicron BA. 2 variant compared to BA. 1 and their possible mouse origins. Cell Research, 1-12.
14. Kumar, S., Karuppanan, K., & Subramaniam, G. (2022). Omicron (BA. 1) and Sub‐Variants (BA. 1.1, BA. 2 and BA. 3) of SARS‐CoV‐2 Spike Infectivity and Pathogenicity: A Comparative Sequence and Structural‐based Computational Assessment. Journal of Medical Virology.
15. Nguyen, H. L., Thai, N. Q., Nguyen, P. H., & Li, M. S. (2022). SARS-CoV-2 Omicron Variant Binds to Human Cells More Strongly than the Wild Type: Evidence from Molecular Dynamics Simulation. The Journal of Physical Chemistry B.
16. Wu, L., Zhou, L., Mo, M., Liu, T., Wu, C., Gong, C., ... & Xu, Z. (2022). SARS-CoV-2 Omicron RBD shows weaker binding affinity than the currently dominant Delta variant to human ACE2. Signal transduction and targeted therapy, 7(1), 1-3.
17. Socher, E., Heger, L., Paulsen, F., Zunke, F., & Arnold, P. (2022). Molecular dynamics simulations of the delta and omicron SARS-CoV-2 spike–ACE2 complexes reveal distinct changes between both variants. Computational and structural biotechnology journal, 20, 1168-1176.
18. Mannar, D., Saville, J. W., Zhu, X., Srivastava, S. S., Berezuk, A. M., Tuttle, K. S., ... & Subramaniam, S. (2022). SARS-CoV-2 Omicron variant: Antibody evasion and cryo-EM structure of spike protein–ACE2 complex. Science, 375(6582), 760-764.
19. Miotto, M., Di Rienzo, L., Gosti, G., Bo, L., Parisi, G., Piacentini, R., ... & Milanetti, E. (2022). Inferring the stabilization effects of SARS-CoV-2 variants on the binding with ACE2 receptor. Communications biology, 5(1), 1-13.
20. da Costa, C. H. S., de Freitas, C. A. B., Alves, C. N., & Lameira, J. (2022). Assessment of mutations on RBD in the Spike protein of SARS-CoV-2 Alpha, Delta and Omicron variants. Scientific Reports, 12(1), 1-10.
21. Kim, S., Liu, Y., Ziarnik, M., Cao, Y., Zhang, X. F., & Im, W. (2022). Binding of human ACE2 and RBD of Omicron enhanced by unique interaction patterns among SARS-CoV-2 variants of concern. bioRxiv.
22. Cui, Z., Liu, P., Wang, N., Wang, L., Fan, K., Zhu, Q., ... & Wang, X. (2021). Structural and functional characterizations of altered infectivity and immune evasion of SARS-CoV-2 Omicron variant. BioRxiv.